indigo_app/templates/indigo_api/_work_form_content.html

```diff
@@ -454,6 +464,18 @@ <h4 class="text-muted work-frbr-uri">{{ work.frbr_uri }}</h4>
       {% endif %}
     {% endblock work-form-repeals-made %}
 
+    {% block work-form-consolidations %}
+      {% if work.pk %}
+        <div class="card mb-3">
+          <div class="card-body">
+            <div id="work-form-repeals-made">
+              {% include 'indigo_api/_work_form_consolidations.html' %}
+            </div>
+          </div>
+        </div>
+      {% endif %}
+    {% endblock work-form-consolidations %}
+
```

Comment:
I would just move this up to be above the first Amendment block

Reply:
('Amended by')

Reply:
@goose-life updated